lib/projects/index.ts

```diff
@@ -325,43 +244,3 @@ export function logFeedbackMessage(buildId: number): void {
     logger.log(i18n(`${i18nKey}.logFeedbackMessage.feedbackMessage`));
   }
 }
-
-export async function createProjectComponent(
```

Comment:
I removed `createProjectConfig` and `createProjectComponent` and put the functionality into the project create and add command files (respectively)